flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/delegation/DefaultExecutor.java

```diff
@@ -55,7 +81,20 @@ public JobClient executeAsync(Pipeline pipeline) throws Exception {
         return executionEnvironment.executeAsync((StreamGraph) pipeline);
     }
 
-    protected String getNonEmptyJobName(String jobName) {
+    private StreamGraph createBatchGraph(
```

Comment:
Is there any reason why we don't push this logic (createBatch* vs createStreaming*) down into `ExecutorUtils`, given that everything we call already lives there as well?

Reply:
This is just an intermediate PR. The methods will be removed in one of the next ones again. `ExecutorUtils` will not be necessary.